src/suparnaNJune20/Assignment_21/MaxNoAndFibonacci.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Assignment 21 : 30th Jun 2020
+
+Program 1: Swap two numbers without using third variable.
+Input: num1= 10, num2 =20
+output: num1=20, num2 =10
+
+Program 2: To print first n Fibonacci Numbers.
+Input n = 6
+Output = 1 1 2  3  5  8
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55
+
+ */
+package suparnaNJune20.Assignment_21;
+
+public class MaxNoAndFibonacci {
```

Comment:
Requirement is to create 2 separate programs.

Reply:
created 2 separate program.